codalab/rest/interpret.py

```diff
@@ -218,8 +230,13 @@ def fetch_interpreted_worksheet(uuid):
                 if not is_relevant_block:
                     interpreted_blocks['blocks'][i] = None
 
+    # items: actual blocks in the worksheets, each item may group several raw items
```

Comment:
Instead of adding these comments inline, can you instead add information about each key in the response (`raw_to_block`, `block_to_raw`, etc.) in the docstring at the top of this function?